### DISCUSSION TITLE
Help Needed Megane 3RS (Ph3) "Error Prog"

### DISCUSSION THREAD

**Discussion by justanegineer** [cedricp/ddt4all#874] [Q&A]
Hi there,

Urgent help needed after a garage used DDT4ALL to reprogram a few things in my car. 

It's a 2015 Megane RS used for track days. 

Their mission was to decode all airbags (front & seat), the seat position sensor and to change the digital speedo in the digital side dash from mph to kph. 

Sadly they have messed up big time resulting in the "Error prog" message. 

I do believe to know the ECUs they have touched, but would love to get some help!

**Comment by matzel82** [comment]
Going to a Renault garage and do a automatic configuration with original Clip.... voila

**Comment by AndyIreland** [comment]
Did they disable ESC? 

what you should do is download PyREN and do DTC errors on the ECU's and clear them ... then see if any errors store again

EDIT: if you have an ELM327 and laptop that is

**Comment by KarelSvo** [comment]
> Sadly they have messed up big time resulting in the "Error prog" message.

In the ddt4all log files show what you have changed. Upload them so that they can be checked.

**Reply by AndyIreland** [reply]
@KarelSvo  if it was a garage that done it for him and they used DDT4ALL then they would have the log files on their laptop so he wont be able to upload the log on here

**Comment by AndyIreland** [comment]
I have turned off TPMS for some people remotely. - If you turn off TPMS but dont turn off the setting in Cluster/TDB you can get "error prog" showing up in the dash cluster - it could be something as simple as that. Once the setting is disabled in the cluster settings if you car is not equipped with it, and you then restart the cluster (with DDT4ALL) then the error goes away.

**Reply by billywilliams2025** [reply]
DDT4all  5.5.0 All cars Edition  

Problem:   ERROR PROG
Object:   To disable TPMS sensor 

Car:  2015 Renault Fluence  Expression 1.5 DCI 110 

Tools: USB OBD Link SX

Using this youtube tutorial  

https://youtu.be/8_MwY3U1klw?si=fj2zEZf7uMUiZuTS

With the choice between:   (See Photo attached)

X38 - FLUENCE 
XFF - FLUENCE 2
XJL - FLUENCE KOREA


I choose XFF- FLUENCE 2

ECU name Cluster_15-40_v3.20.00
Cluster/TDB
Ecrans Configuration
Conf Prestations IHM

I changed from  “WITH TO WITHOUT”

Configuration Now:      TPMSMMI_CF   -   Without TPMS

And Reset Dashboard as per instruction from youtube tutorial


![CHECKTYRE SENSOR](https://github.com/user-attachments/assets/85b6647f-fed5-48ed-95e2-e9eba70e97f6)
![Cluster_15-40_v3 30 05](https://github.com/user-attachments/assets/d0190824-ad46-42de-b9ee-35213fec6479)
![Conf Orestations IHM](https://github.com/user-attachments/assets/70f922b2-b564-43f8-b804-c1b8b4fe0a6f)
[Ecrans Configuration.pdf](https://github.com/user-attachments/files/19053870/Ecrans.Configuration.pdf)
![ERRORPROG](https://github.com/user-attachments/assets/22073d3e-d176-4ed3-9b8f-3b434ae3024c)
![TDB](https://github.com/user-attachments/assets/ec7c364f-f302-4c18-8a95-3cc04cffb369)
![XFFFLUENCE2](https://github.com/user-attachments/assets/e640c4a9-2bf5-4ae4-82c2-2704be23721b)

I took a screenshot of Cluster_15-40_v3.20.05 that also has the settings changed from WITH TO WITHOUT TPMS although I did not carry out any changes in this cluster.

Any help is much appreciated, thanks.

**Comment by AndyIreland** [comment]
they could have even disabled your airbags but not changed the setting in the cluster tdb to without airbag ecu? 
![image](https://github.com/cedricp/ddt4all/assets/49172217/67806c29-9ab9-4771-b7a4-dbe41e70dfdc)

**Comment by shrlnm** [comment]
Hello @justanegineer
You may revert any changes by this instruction https://www.drive2.ru/b/643355315688994881/
